config/common/common.views.xml

```diff
@@ -2634,32 +1944,55 @@
 					<cell type="label" labelfor="2"/>
-					<cell type="field" id="2" name="isSubstrate" uitype="checkbox"/>
+					<cell type="field" id="2" name="isSubstrate" uitype="checkbox" default="false"/>
 					<!--<cell type="label" labelfor="3"/>
       <cell type="field" id="3" name="precedence" uitype="text"/>-->
 					<cell type="label" labelfor="5"/>
-					<cell type="field" id="5" name="childCog" uitype="querycbx" readonly="true"/>
+					<cell type="field" id="5" name="childCog" uitype="querycbx" readonly="true" colspan="3"/>
 					<cell type="label" labelfor="6"/>
-					<cell type="field" id="6" name="childCo" uitype="querycbx" readonly="true"/>
+					<cell type="field" id="6" name="childCo" uitype="querycbx" readonly="true" colspan="3"/>
 				</row>
 				<row>
-					<cell type="panel" colspan="15">
+					<cell type="separator" label="Child Attributes" colspan="20"/>
+				</row>
+				<row>
+					<!-- Collection Object Group Details -->
+					<cell type="label" label="COG Type"/>
+					<cell type="field" id="typeOfGroup" name="childCog.COGType.Name" uitype="text" readonly="true"/>
+					<!-- Collection Object Details -->
+					<cell type="label" label="CO Type"/>
+					<cell type="field" id="objectType" name="childCo.collectionObjectType.Name" uitype="text" readonly="true"/>
+					<cell type="label" label="Name"/>
+					<cell type="field" id="currentDet" name="childCo.currentDetermination.preferredTaxon.fullName" uitype="text" readonly="true"/>
+					<cell type="label" label="Collecting Event"/>
+					<cell type="field" id="collectingEvent" name="childCo.collectingEvent.stationFieldNumber" uitype="text" readonly="true"/>
+					<cell type="label" label="Locality"/>
+					<cell type="field" id="locality" name="childCo.collectingEvent.locality.localityName" uitype="text" readonly="true"/>
+				</row>
+				<row>
+					<cell type="panel" colspan="20">
 						<rows>
 							<row/>
 							<!-- Show nothing if no Collection Object Group (COG) or Collection Object (CO) child exists for this COJO record -->
 						</rows>
-						<rows condition="childCog.CollectionObjectGroupID=">
-							<!-- If there is no CollectionObjectGroupID, display CO -->
+						<rows condition="childCo.collection.ID=4">
 							<row>
-								<cell type="subview" viewname="CollectionObjectSub" id="childCO" name="childCo" uitype="subview" colspan="13"/>
+								<cell type="subview" viewname="CollectionObjectMini" id="childCO" name="childCo" uitype="subview" colspan="20"/>
 							</row>
 						</rows>
-						<rows condition="childCo.CollectionObjectID=">
-							<!-- If there is no CollectionObjectID, display COG -->
+					</cell>
+				</row>
+				<row>
+					<cell type="panel" colspan="15">
+						<rows>
+							<row/>
+							<!-- Show nothing if no Collection Object Group (COG) or Collection Object (CO) child exists for this COJO record -->
+						</rows>
+						<rows condition="childCog.collection.ID=4">
```

Comment:
Hardcoded id here as well?

Reply:
Fixed in [`1805b60` (#5367)](https://github.com/specify/specify7/pull/5367/commits/1805b60eab4cb7f20798dd561ff299e7eb1f0f71)